src/lib/tasks/run-removal.ts

```diff
@@ -124,42 +116,48 @@ export function runRemoval(
         continue;
       }
 
-      // The sprite has been marked for removal, its in the right
+      // The sprite has been marked for removal, it's in the right
       // LifecyclePhase, and its time has expired. Flash zeros to the sprite's
       // data view and schedule it for a texture sync.
       properties.spriteView[DataViewSymbol].fill(0);
+      properties.zeroed = true;
```

Comment:
For #77, signals that a sprite that has been marked for removal has had its values flashed to zero.